lingetic-nextjs-frontend/posthog.ts

```
@@ -0,0 +1,15 @@
+import { PostHog } from "posthog-node";
+import assert from "./utilities/assert";
+
+export default function PostHogClient() {
+  assert(
+    process.env.NEXT_PUBLIC_POSTHOG_KEY != null,
+    "PostHog key is required"
+  );
+  const posthogClient = new PostHog(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)